src/Elmish/HTML.purs

```diff
@@ -4,6 +4,6 @@ module Elmish.HTML
     , module ExportedInternal
     ) where
 
-import Elmish.React.DOM (text) as ExportedDOM
+import Elmish.React.DOM (text, empty, fragment) as ExportedDOM
```

Comment:
Added reexports of `empty` and `fragment` as a drive-by improvement - something I wanted to do for a while.